### DISCUSSION TITLE
childTransaction vs transaction

### DISCUSSION THREAD

**Discussion by janekolszak** [kriszyp/lmdb-js#209] [Q&A] [answered]
Hi!
Is a childTransaction supposed to be used only inside transaction or can it be used standalone?
Thank you!

**Comment by kriszyp** [comment, marked as answer]
It can be used standalone (as an asynchronous transaction), and can be useful on its own because it is abortable (by returning ABORT or throwing), whereas the standard asynchronous `transaction` method is not abortable.

**Reply by janekolszak** [reply]
And a `childTransaction` inside a `childTransaction` is OK I presume?
